src/main/java/pl/miloszkajetan/serwisAukcyjny/User/UserRoleRepository.java

```diff
@@ -0,0 +1,14 @@
+package pl.miloszkajetan.serwisAukcyjny.User;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface UserRoleRepository extends JpaRepository<UserRole, Long> {
+
+    @Query("SELECT ur FROM UserRole ur WHERE ur.userRoleName = '1' ")
```

Comment:
Z tym Query nie jesteśmy pewni, czy ta '1' jako indeks jest poprawnie, czy powinniśmy mimo wszystko zrobić to inaczej?

Reply:
raczej ?1 jeżeli chcecie w ten sposób